source/postprocess/geoid.cc

```diff
@@ -931,9 +1015,16 @@ namespace aspect
       {
         prm.enter_subsection("Geoid");
         {
+          // This parameter doesn't exist anymore (incl. spelling error). TODO: throw error message when used.
```

Comment:
No need to have this comment here. This line only tells ASPECT that it is ok to find this parameter in the input file.